receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -27,6 +27,19 @@ query getRepoDataBySearch(
         defaultBranchRef {
           name
         }
+        vulnerabilityAlerts(first: 100, states: OPEN) {
```

Comment:
Did we want to incorporate this in the `getRepoDataBySearch` query, or did we want to make a separate query. As it stands, the feature flag is useless given that this is already being incorporated into a query that runs.

Reply:
It was completely separated.